ui/app/controllers/evaluations/index.js

```diff
@@ -35,7 +35,7 @@ export default class EvaluationsController extends Controller {
     'currentEval',
     'pageSize',
     'status',
-    'qpNamespace',
+    { qpNamespace: 'namespace' },
```

Comment:
`blocking`: I don't think this resolves the issue. This sets the alias of the query parameter in the URL. But, this shouldn't impact formatting the request in the route.

![image](https://user-images.githubusercontent.com/41024828/176549623-85ae7ad9-7987-465a-9bff-9d70ee07de7f.png)

If we need to make any edits, I would suspect we'd need to check the formatting logic in `generateFilterExpression`. However, the [tests](tests/acceptance/evaluations-test.js) assert on how user actions impact the request we're sending and the default request we send when we enter the view. I'd love to learn if we wrote those tests and the formatting logic incorrectly.

Reply:
That's your call to make. We can totally adopt that pattern, but that's an undocumented behavior we'd have to load into our mental models. We specifically designed this to set namespace to `*` when entering this view. I could see the mental baggage of keeping this type of knowledge in our heads with an Ember Routing API change.

It's good UX to have. So totally your call, I'll support whatever you decide. I'll update my review on this PR to approved and let you handle this how you think is best.